AsyncDisplayKit/TextKit/ASTextKitRenderer+TextChecking.mm

```diff
@@ -75,6 +74,7 @@ - (NSTextCheckingResult *)textCheckingResultAtPoint:(CGPoint)point
     truncationTokenRange = { 0, truncationAttributedString.length };
   }
 
+  NSRange visibleRange = self.truncater.firstVisibleRange;
   truncationTokenRange.location += NSMaxRange(visibleRange);
```

Comment:
I'm open to change, just thinking it makes more sense this way and aligns a bit more with Apple APIs. For example:

[NSRangeFromString](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Miscellaneous/Foundation_Functions/Reference/reference.html#//apple_ref/doc/uid/20000055-BAJBHBDJ): Returns a range from a textual representation.
> ... If aString does not contain any integers, this function returns an NSRange struct whose location and length values are both 0.

[-[NSString rangeOfString:]](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSString_Class/Reference/NSString.html#//apple_ref/occ/instm/NSString/rangeOfString%3a): Here you pass in the string you search for and it returns NSNotFound as location. That's imho pretty similar to `firstVisibleRange`
> ... Returns {NSNotFound, 0} if aString is not found or is empty (@"").